apps/api/infra/api.ts

```diff
@@ -40,7 +40,7 @@ export const serviceSetup = (services: {
         prod: 'island-is-prod-upload-api',
       },
       AUTH_PUBLIC_API_URL: {
-        dev: 'https://identity-server.dev01.devland.is',
+        dev: 'https://identity-server.dev01.devland.is/public',
```

Comment:
why only for dev and didn't you forget to run `yarn charts`?

Reply:
Yes I forgot to run yarn charts.

I'm testing the ingress rewrite-target rule on dev by manually configuring in k9s.
Then the plan is to change this path to `/api` and do the corresponding changes in the Helm repo for the identity-server apis.

Reply:
ah right so this is work in progress. you know you can create draft pull requests, and then there will be no reviewers. kind of nice when you're playing a :ninja: